exercises/basiccalcobject.js

```diff
@@ -1,18 +1,37 @@
 
 class Calculator{
 	constructor(){
+		this.numbers = [];
+		this.op = null;
 		//takes in nothing
 		//make storage for the operator and the numbers
 	}
-	loadOperator(  ){
+	loadOperator(operator){
+		var possibleOperators = ['+','-','*','/']
+		if (possibleOperators.includes(operator)){
+			this.op=operator;
+			return true;
+		} else {
+			return false;
+		}
 		//adds the operator to the next calculation
 		//takes in the operator
 		//checks if it is a valid operation (+-*/)
 			//save the op to the constructor storage
 			//return true
 		//or return false if not the right operator
 	}
-	loadNumber(  ){
+	loadNumber(num){
+		if (typeof num === 'number') {
+			if (this.numbers.length<2){
+				this.numbers.push(num)
+				return this.numbers.length;
+			} else {
+				return false
+			}
+		} else {
```

Comment:
since really at this point, either else you are returning false, you could have gotten rid of both elses and just returned false at the end of the function, since you were returning a value if it worked.